library/common-library/templates/_cluster.tpl

```diff
@@ -0,0 +1,17 @@
+{{/*
+Return the cluster
+*/}}
+{{- define "common.cluster" -}}
+    {{- if .Values.cluster -}}
+        {{- .Values.cluster -}}
+    {{- else if .Values.global -}}
+        {{- if .Values.global.cluster -}}
+            {{- .Values.global.cluster -}}
+        {{ else }}
+            {{- "" -}}
```

Comment:
Do we need these `else ""` constructs? In my past experience it was fine to simply omit them, Helm will render nothing by default which should be evaluated to `false`.

Reply:
The helper will return an empty string if you print nothing. This is redundant. No helper has those so I would remove this.

Reply:
oks, I will remove it

Reply:
done